releasenotes/notes/change-tool-call-output-openai-anthropic-f59f6ef816589cf8.yaml

```
@@ -0,0 +1,7 @@
+---
+fixes:
```

```suggestion
features:
```
Since this is a new feature, not a fix